dvc/progress.py

```diff
@@ -151,4 +151,14 @@ def finish(self, name):
         progress.finish_target(name)
 
 
+def progress_wrapper(iterable):
```

Comment:
Isn't `Progress.__call__` the same thing? E.g. https://github.com/iterative/dvc/blob/master/dvc/remote/base.py#L148

Reply:
:eyes: ! It is, @efiop , good catch